packages/frontend/src/components/campaign-creation-form.tsx

```diff
@@ -141,39 +163,50 @@ export function CampaignCreationForm<S extends SerializableObject<S>>() {
         invalidateLatestKPITokens();
     }, [invalidateLatestKPITokens]);
 
+    const handleCreateDraft = useCallback(() => {
+        if (!template) {
+            console.log("couldn't create draft, missing template id");
+            return;
+        }
+        addDraft(currentDraftId, template?.id, draftState.state);
+    }, [currentDraftId, addDraft, draftState, template]);
+
     return loading ? (
         <div className="h-screen py-20 text-black flex justify-center">
             <Loader />
         </div>
     ) : template ? (
-        <KPITokenCreationForm
-            key={formKey}
-            template={template}
-            fallback={
-                <div className="h-screen py-20 text-black flex justify-center">
-                    <Loader />
-                </div>
-            }
-            error={
-                <div className="h-screen py-20 flex justify-center">
-                    <ErrorFeedback
-                        messages={{
-                            title: t("error.initializing.creation.title"),
-                            description: t(
-                                "error.initializing.creation.description",
-                            ),
-                        }}
-                    />
-                </div>
-            }
-            i18n={i18n}
-            className={{ root: "w-full h-full" }}
-            state={draftState.state}
-            onStateChange={handleStateChange}
-            onCreate={handleCreate}
-            navigate={navigate}
-            onTx={addTransaction}
-        />
+        <>
```

Comment:
Why is there a wrapping React fragment here?

Reply:
Fixed in eb18f5629b3ac453997b41920f523ab418f1787e.